### PR TITLE
ACMS-1275: Add CI/CD Pipeline to ACMS Audio module.

### PR DESCRIPTION
**Pull request by rajeshreeputra**

### CODE REVIEW

File: .github/workflows/acquia_cms_audio.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        if: ${{ success() }}
+        run: ../orca/bin/ci/after_success.sh
+
+      # Display debugging information in case of failure.
+      - name: After failure
+        if: ${{ failure() }}
+        run: ../orca/bin/ci/after_failure.sh
+  VERIFY_ACMS_INSTALL_ON_DRUPAL_PROJECT:
+    if: ${{ github.event_name == 'push' }}
+    name: "Verify commands on acquia/drupal-recommended-project"
```
Comment (chandan-singh7929) [inline .github/workflows/acquia_cms_audio.yml:80]: Change this to "Verify acquia_cms_audio on acquia/drupal-recommended-project

File: .github/workflows/acquia_cms_audio.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      # Conditionally send code coverage data to Coveralls.
+      - name: After success
+        if: ${{ success() }}
+        run: ../orca/bin/ci/after_success.sh
+
+      # Display debugging information in case of failure.
+      - name: After failure
+        if: ${{ failure() }}
+        run: ../orca/bin/ci/after_failure.sh
+  VERIFY_ACMS_INSTALL_ON_DRUPAL_PROJECT:
```
Comment (chandan-singh7929) [inline .github/workflows/acquia_cms_audio.yml:78]: Update Job name accordingly.

**Review by chandan-singh7929 — APPROVED** [review]
Overall looks good to me, have some minor comment, please check.
Also, update the PR title with correct module name.

File: .github/workflows/acquia_cms_audio.yml (hunk 1 above)
Comment (rajeshreeputra) [reply .github/workflows/acquia_cms_audio.yml:80]: updated

File: .github/workflows/acquia_cms_audio.yml (hunk 2 above)
Comment (rajeshreeputra) [reply .github/workflows/acquia_cms_audio.yml:78]: updated

**Pull request merged**